integration/src/test/java/org/apache/iotdb/db/integration/IoTDBTtlIT.java

```diff
@@ -77,6 +77,31 @@ public void testTTL() throws SQLException, InterruptedException {
         assertEquals(TSStatusCode.STORAGE_GROUP_NOT_EXIST.getStatusCode(), e.getErrorCode());
       }
 
+      statement.execute("SET STORAGE GROUP TO root.TTL_SG2");
+      statement.execute("CREATE TIMESERIES root.TTL_SG2.s1 WITH DATATYPE=INT64,ENCODING=PLAIN");
+      try {
+        statement.execute("SET TTL TO root.TTL_SG2.s1 1000");
+      } catch (SQLException e) {
+        assertEquals(TSStatusCode.STORAGE_GROUP_NOT_EXIST.getStatusCode(), e.getErrorCode());
+      }
+
+      try {
+        statement.execute("SET TTL TO root.** 1000");
+      } catch (SQLException e) {
+        assertEquals(TSStatusCode.TIMESERIES_NOT_EXIST.getStatusCode(), e.getErrorCode());
+      }
```

Comment:
The IT in integration package is only run by old-standalone. Add the IT to integration-test package and follow the new-IT rule. Besides, if you expect exception in this test case, ```Assert.fail()``` is necessary.

Reply:
Okay, got it.